test/helm/default-config.out.yaml

```diff
@@ -1,5 +1,19 @@
 ---
 # Source: gateway-helm/templates/envoy-gateway-deployment.yaml
+apiVersion: policy/v1
```

Comment:
this seems unreasonable, PDB should be opt-in.

Reply:
With the control plane, it can be added, but by default, it won’t take effect because the minimum pods available is set to zero. So effectively, it won’t enforce anything. wdyt?